### PR TITLE
Fix touch release event on _gui_input

### PR DESCRIPTION
**Pull request by lethiandev**
Fixes https://github.com/godotengine/godot/issues/16761

Last touch release event is properly called on focused control now.

### CODE REVIEW

**Comment by akien-mga** [comment]
CC @RandomShaper

**Comment by lethiandev** [comment]
Ok, done, conflicts are resolved now.

File: scene/main/viewport.cpp [hunk 1]
```diff
@@ -1727,6 +1727,8 @@ void Viewport::_gui_input_event(Ref<InputEvent> p_event) {
 
 				gui.mouse_focus = _gui_find_control(pos);
 				gui.mouse_focus_mask = 1 << (mb->get_button_index() - 1);
+				gui.mouse_focus_button = mb->get_button_index();
```
Comment (akien-mga) [inline scene/main/viewport.cpp]:
```
scene/main/viewport.cpp:1730:9: error: 'struct Viewport::GUI' has no member named 'mouse_focus_button'
     gui.mouse_focus_button = mb->get_button_index();
         ^
```
This was removed in a previous commit, it should not be readded.

**Comment by lethiandev** [comment]
Thanks for pointing it out, should be fixed 😄

**Pull request merged**